src/components/Header/HeaderCart/HeaderCart.jsx

```diff
@@ -0,0 +1,63 @@
+import useWindowWidth from "../../../../hooks/useWindowResize.js";
+import CartList from "./CartList/CartList.jsx";
+import './HeaderCart.css';
+import CartImg from "../../../../public/img/cart.png";
+
+export default function HeaderCart(props) {
+    const {
+        headerCartProducts,
+        setHeaderCartProducts,
+        setIsOpenHeaderCartModal,
+        isOpenHeaderCartModal,
+        isCartShakingAnimation,
+        setIsCartShakingAnimation,
+    } = props;
+
+    const productQuantity = headerCartProducts.length;
+    let getEachProductSum = headerCartProducts.map(product => product.sum);
+    let getAllProductSum  = getEachProductSum.reduce((sum, current) => sum + current, 0);
```

Comment:
шо за лишній пробіл?

Reply:
Лишній пробіл не поправив же. Натомість ти просто видалив пустий рядок. Верни його назад
```suggestion
    let getAllProductSum = getEachProductSum.reduce((sum, current) => sum + current, 0);
```

Reply:
@viktor-fedoruk який вже раз пишу, 4ий, 5ий? 
Де видалений пробіл? Я вже картинкою відкрито показував що конкретно треба видалити. 

let getAllProductSum = getEachProductSum.reduce((sum, current) => sum + current, 0);